run.sh

```diff
@@ -8,13 +8,45 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-if [ "$1" = "-prod" ]; then
-  echo "Building for production (publishing on eclipse.org/che/docs/). To preview"
-  echo -e "the docs locally, run the script ($0) without the '-prod' option.\n"
-  docker run --rm -v $(pwd)/src/main:/che-docs:Z eclipse/che-docs sh -c "cd /che-docs; jekyll clean; jekyll build --config _config.yml,_config-war.yml"
-else
-  echo "Building and serving the docs for local preview. To build for production"
-  echo "(publishing on eclipse.org/che/docs/), run the script with the '-prod' option:"
-  echo "$0 -prod"
-  docker run --rm -ti -p 35729:35729 -p 4000:4000 -v $(pwd)/src/main:/che-docs:Z eclipse/che-docs sh -c "cd /che-docs; jekyll clean; jekyll serve --livereload -H 0.0.0.0"
-fi
+set -e
+
+# Detect presence of podman. Fallback to docker
+RUNNER="$(command -v podman 2>/dev/null || command -v docker)"
+
+case "${RUNNER}" in
+  *podman)
+    echo "Using $RUNNER."
+    [ "$(find src/main/_site -user root -print -prune -o -prune 2>/dev/null)" ] && \
+      echo "Previous build was probably created with docker. We need root privileges to delete it." && \
+      sudo rm -rf src/main/_site/ 
+    ;;
+  *docker)
+    echo "podman not detected. Using fallback runner $RUNNER. This runner is deprecated, please install podman."
```

Comment:
I'm not sure one runner should deprecate the other

Reply:
Podman does not write files as the root user, I believe it is a good enough reason to use podman when it is present, and docker only as a fallback.

Maybe this wording would be better, posponing the deprecation of docker:
```suggestion
    echo "The preferred runner podman is not installed. Using fallback runner $RUNNER."
```